validation/validation.go

```diff
@@ -16,10 +16,15 @@ func Validate(cfg *config.Local, tracker issuetracker.IssueTracker) []error {
 		errs = append(errs, err)
 	}
 
-	if err := validateAuthOfflineURL(cfg); err != nil {
-		errs = append(errs, err)
+	if cfg.Auth.Type == config.AuthTypeOffline{
```

Comment:
Could you setup `gofmt` locally so that we avoid the minor issues around formatting such as lacking a space between `AuthTypeOffline` and the curly bracket?

I've also opened a new task to automate this - #138. I think you might find it interesting if you've not dealt with CI setups before :)

Reply:
I included gofmt on goland. I used it on visual code but made the change to goland recently and forgot about adding it on the filewatchers. We won't face another issue with this :)